Documentation/observability/metrics.rst

```diff
@@ -149,6 +153,8 @@ have it scrape all Hubble metrics from the endpoints automatically:
             regex: (.+)(?::\d+);(\d+)
             replacement: $1:$2
 
+.. _hubble_open_metrics:
```

Comment:
I'm somewhat confused by this section. It sounds a bit like OpenMetrics == exemplars, while Hubble metrics without exemplars enabled are OpenMetrics-compatible. It would be helpful IMO to make it clearer what changes (if anything) when `hubble.metrics.enableOpenMetrics=true` is set, but `exemplars` option is not enabled.

Reply:
I'm not sure we need to explain to users what OpenMetrics is, we link to the upstream docs which covers it quite clearly. The important point is that you need to enable openMetrics to use examplars. Do you believe that is not sufficiently covered?

Reply:
Yeah, a link to the upstream docs is enough, and it's clear that exemplars require OpenMetrics enabled. What confused me a bit is this part:
```
Prometheus (...) will only use OpenMetrics format when the exemplars storage feature is enabled
```
I thought Prometheus uses OpenMetrics internally if metrics are compatible, regardless of whether it has exemplars storage or not? If exemplars are not enabled, this doesn't matter from Hubble perspective. Just this part sounded to me like OpenMetrics == exemplars.

Reply:
I see. That quote was basically something I copied from the Prometheus docs here: https://prometheus.io/docs/prometheus/latest/feature_flags/#exemplars-storage. 

Basically, you need to configure `--enable-feature=exemplar-storage` in Prometheus, which will tell it to do content negotiation when scraping metrics. With Exemplars storage enabled, if the metrics endpoint supports OpenMetrics, then Prometheus will scrape via OpenMetrics format.

Reply:
I added a small suggestion. The docs are clear to me overall, just the format negotiation is a Prometheus detail, so I'd try to avoid confusion about it here.